src/main/java/com/alkemy/ong/configuration/SecurityConfig.java

```diff
@@ -42,10 +50,21 @@ public AuthenticationManager authenticationManagerBean() throws Exception {
         return super.authenticationManagerBean();
     }
 
+    @Bean
+    public UserDetailsService userDetailsService(){
+        return (UserDetailsService) email -> {
```

Comment:
It looks great! You can improve it in one line using Optional's orElseThrow method, like: 

```java
return email ->  userService.findUserByEmail(email).orElseThrow(UserNotFoundException::new)
```